content/en/docs/concepts/security/index.md

```diff
@@ -617,9 +615,12 @@ services in an Istio Mesh. It features:
     >}}
 
 The above diagram shows the basic Istio authorization architecture. Operators
-specify Istio authorization policies using `.yaml` files. Once deployed, Istio
-saves the policies in the `Istio Config Store`.
+specify Istio authorization policies using `.yaml` files.
 
+Galley watches for changes to Istio authorization policies. It fetches the
+updated authorization policies if it sees any changes and pushes to Pilot. Pilot
+converts the authorization policies to Envoy filter configuration and distributes the filter
+to the Envoy proxies that are co-located with the workloads.
 Pilot watches for changes to Istio authorization policies. It fetches the
 updated authorization policies if it sees any changes. Pilot distributes Istio
```

Comment:
Pilot watches for changes to Istio authorization policies~.~ and ~It~ fetches ~the~
updated authorization policies if it sees ~any~ changes.